blueutil.m

```diff
@@ -301,6 +420,81 @@ bool parse_signed_long_arg(char *arg, long *number) {
   }
 }
 
+NSArray *get_paired_devices_subprocess() {
+  NSTask *task = [[NSTask alloc] init];
+  task.launchPath = @"/usr/sbin/system_profiler";
+  task.arguments = @[@"SPBluetoothDataType", @"-xml"];
+
+  NSPipe *pipe = [NSPipe pipe];
+  task.standardOutput = pipe;
+  task.standardError = [NSPipe pipe];
+
+  [task launch];
+  [task waitUntilExit];
+
+  NSData *data = [[pipe fileHandleForReading] readDataToEndOfFile];
+  if (task.terminationStatus != 0 || data.length == 0) {
```

Comment:
I'm thinking that this should probably abort `blueutil` if termination status is not success